### ISSUE TITLE
Unable to bookmark blogs and updates listing

### ISSUE THREAD

**Issue by markcellus** [microsoft/vscode#168481]
Does this issue occur when all extensions are disabled?: N/A

- VS Code Version: N/A
- OS Version: N/A

Steps to Reproduce:

1. Navigate to blogs listing page (https://code.visualstudio.com/blogs)
2. Bookmark page
3. Verify bookmarked page is not https://code.visualstudio.com/blogs but to the latest article (https://code.visualstudio.com/blogs/2022/12/07/remote-even-better at the time of this writing)

This also happens when navigating to https://code.visualstudio.com/updates.

**Comment by andreamah** [comment]
I can't repro this. Is this only happening with https://code.visualstudio.com websites?

**Comment by markcellus** [comment]
I'm just using a browser. What URL does clicking https://code.visualstudio.com/blogs take you to? For me, it redirects the URL to https://code.visualstudio.com/blogs/2022/12/07/remote-even-better.

**Comment by andreamah** [comment]
Ohh, I see now. So you expect https://code.visualstudio.com/blogs to show a list of blog posts, but it takes you straight to the remote post?

**Comment by markcellus** [comment]
Exactly. So there's no way to bookmark (or even share) a URL listing all of posts.

**Comment by gregvanl** [comment]
Hi @markcellus There is an RSS feed listing the releases and blog posts at https://code.visualstudio.com/feed.xml, which might support your scenario.

It's by design that the site takes you to the latest release notes and blog post. The left navigation list has each update or blog post.

**Comment by markcellus** [comment]
Thanks @gregvanl but see my replies on this thread. The RSS feed isnt what I'm asking here. I'm asking for a way to bookmark or share a URL to the full list if blog posts.

**Comment by gregvanl** [comment]
I thought you might want this for some sort of automation. 
A reader can see a listing of all updates or blog posts in the left navigation.

**Comment by markcellus** [comment]
The left nav is useful but only when you've already *intentionally* navigated to the URL of a specific blog entry. 

Sorry but why would a person want to be auto redirected to some arbitrary entry when the person just wants to see a listing of all the blog post written? It's very odd that this happens. I've never seen a blog route like this before.

**Comment by gregvanl** [comment]
If I understand, you want to landing page listing the blogs posts rather than showing the most recent with a list on the left?

Bookmarking https://code.visualstudio.com/blogs works but it isn't showing you a general list landing page

**Comment by gregvanl** [comment]
Moving to the vscode-docs repository since this is not a product issue.
